manifests/repos.pp

```diff
@@ -54,6 +54,10 @@
         }
       }
 
+      'Suse': {
+        # Suse nutzt archive, kein repo
+      }
```

Comment:
Is this manifest actually included when repo management is disabled? I would think that on SUSE this should fail since we are not managing the repo.

Reply:
I am going to disable repo management on sles using hiera data and remove the sles lines from puppet code.